### ISSUE TITLE
GM 4.5 does not work in Torbrowser 8.0a9

### ISSUE THREAD

**Issue by LRN** [greasemonkey/greasemonkey#2979]
When installed onto Torbrowser 8.0a9 (based on Mozilla Firefox 60.1.0esr) (32-bit) GM 4.5 does not work.
The "New user script..." menu item does nothing when clicked.
This is what the browser console says:
```
12:19:48.940 Error opening user-scripts DB! <unavailable>  user-script-registry.js:57:7
	openDb/</dbOpen.onerror moz-extension://dfaa9521-db7d-477d-ad92-1a305e431259/src/bg/user-script-registry.js:57:7
12:19:48.945 <unavailable>  downloader.js:111:11
	install/</< moz-extension://dfaa9521-db7d-477d-ad92-1a305e431259/src/downloader.js:111:11
12:19:48.946 Error: undefined  downloader.js:105:7
```
Uninstalling, re-running TB and installing GM again didn't work.
Renaming the `gm_scripts` directory to ensure that GM considers it non-existing didn't work also.

**Comment by johnp** [comment]
Sounds like [bug 781982](https://bugzilla.mozilla.org/show_bug.cgi?id=781982), i.e. IndexedDB does not work in Private Mode and Tor Browser launches by default into Private Mode.

**Comment by Thorin-Oakenpants** [comment]
FYI: TBB also has `privacy.firstparty.isolate` defaulting to true, so #2985 will also have an affect

**Comment by jbostoen** [comment]
Good to see the workaround, can confirm it's still a bug with the now officially released Tor Browser 8.0.1
